pyproject.toml

```diff
@@ -52,69 +52,57 @@ version = "0.1.1"
 [tool.poetry.dependencies]
 bidict = "^0.23.1"
 numba = [
-  {python = ">=3.9,<4.0", version = "^0.60.0"},
-  {python = "3.8", version = "^0.58.0"}
+  {python = ">=3.9,<3.13", version = "^0.60.0"}
```

Comment:
This shouldn't have a python restriction anymore, why not just use this PR to add 3.13 support?